scripts/error-codes/codes.json

```diff
@@ -515,5 +515,10 @@
   "527": "Incompatible React versions: The \"react\" and \"react-dom\" packages must have the exact same version. Instead got:\n  - react:      %s\n  - react-dom:  %s\nLearn more: https://react.dev/warnings/version-mismatch",
   "528": "Expected <link> not to update to be updated to a stylesheet with precedence. Check the `rel`, `href`, and `precedence` props of this component. Alternatively, check whether two different <link> components render in the same slot or share the same key.%s",
   "529": "Expected stylesheet with precedence to not be updated to a different kind of <link>. Check the `rel`, `href`, and `precedence` props of this component. Alternatively, check whether two different <link> components render in the same slot or share the same key.%s",
-  "530": "The render was aborted by the server with a promise."
+  "530": "The render was aborted by the server with a promise.",
+  "531": "react-html is not supported outside a React Server Components environment.",
+  "532": "Attempted to render a Client Component from renderToMarkup. This is not supported since it will never hydrate. Only render Server Components with renderToMarkup.",
```

Comment:
This may block people from adopting. E.g. you could render client components with `renderToStaticMarkup` and were fine with it not being hydrate. `renderToMarkup` throwing may require a non-trivial refactor. But let's ship and see.

Reply:
Yea.

1) We don't have an option that doesn't involve also allowing them for RSC. Clearly it's a way too easy mistake to import an unmarked Client Component in a Server Component and have it no longer work.
2) The same mistake applies to e-mails too so it's a good thing.